src/anls_star.py

```diff
@@ -277,7 +311,21 @@ def _levenshtein_distance(s1: str, s2: str):
         return distances[-1]
 
 
-def anls_score(gt, pred):
+@overload
+def anls_score(gt: Any, pred: Any, *, return_gt: Literal[False]) -> float: ...
+
+
+@overload
+def anls_score(
+    gt: Any, pred: Any, *, return_gt: Literal[True]
+) -> tuple[float, Any]: ...
+
+
+@overload
+def anls_score(gt: Any, pred: Any, *, return_gt: bool = False) -> float: ...
```

Comment:
```suggestion
def anls_score(gt: Any, pred: Any, *, return_gt: bool = False) -> Union[float, tuple[float, Any]]: ...
```
I think that is the correct overload in this case. If `return_gt` is of type `bool` (the `default = False` does not matter I think), we don't know what the output will be, so the output type should be the union of both options.